data_architect_misc/data_transformer/transform_functions/latam_mexico_transform_functions.py

```diff
@@ -0,0 +1,92 @@
+"""This is the subclass of Transform function for Mexico (LATAM division).
+
+We will define transform functions specific to Mexico here.
+
+Author: Maicol Contreras
+Last Modified: September 9, 2020
+"""
+
+import pandas as pd
+
+from constants import comp_harm_constants
+from transform_functions.common_comp_harm_transform_functions import CommonCompHarmTransformFunctions
+
+
+class LatamMexicoTransformFunctions(CommonCompHarmTransformFunctions):
+    """
+    All custom (uncommon) transform functions **SPECIFIC to
+    individual processing task** must be defined as part
+    of this class.
+    """
+    MEXICO_SPECIFIC_CATEGORY_MAPPINGS = {
```

Comment:
Don Maicol, 

Will any of these category mappings be applicable for other LATAM countries? If so, why don't we merge them into 'SPANISH_CATEGORY_MAPPINGS' in comp_harm_constants.py?